gradle.properties

```diff
@@ -32,8 +32,8 @@ kotlin.native.binary.memoryModel=experimental
 #kotlinx.atomicfu.enableJsIrTransformation=true
 
 kotlin_version=1.8.10
-coroutines_version=1.6.4
-atomicfu_version=0.20.0
+coroutines_version=1.7.0-RC
```

Comment:
1.7.0 is released

Reply:
Raised https://github.com/ktorio/ktor/pull/3609, which contains more things than the automatic one.